ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/cucushift-installer-rehearse-aws-ipi-ovn-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
```

Comment:
No, I think @lihongan is doing right things. we need provision/deprovision chains in the workflow.

Reply:
@jianlinliu ok,Thanks. 
I have another question. both provision and deprovision are must in one feature/folder.  I saw same chains just different file name in different directory.  like below: 

```
# cat deprovision/cucushift-installer-rehearse-vsphere-ipi-deprovision-chain.yaml 
chain:
  as: cucushift-installer-rehearse-vsphere-ipi-deprovision
  steps:
  - chain: gather
  - ref: ipi-deprovision-vsphere-diags
  - ref: ipi-deprovision-deprovision
  - ref: ipi-deprovision-vsphere-dns
  - ref: ipi-deprovision-vsphere-lb
  documentation: |-
    Destroy an IPI cluster on vSphere
```
```
# cat disconnected/deprovision/cucushift-installer-rehearse-vsphere-ipi-disconnected-deprovision-chain.yaml 
chain:
  as: cucushift-installer-rehearse-vsphere-ipi-disconnected-deprovision
  steps:
  - chain: gather
  - ref: ipi-deprovision-vsphere-diags
  - ref: ipi-deprovision-deprovision
  - ref: ipi-deprovision-vsphere-dns
  - ref: ipi-deprovision-vsphere-lb
  documentation: |-
    Destroy an IPI cluster in disconnected network on vSphere.
```
so above two chains are same. do we need to create two chains? can we reuse it in another workfolw?

Reply:
The contents are the same, that is because the cluster destroy step are the same (destroy is easier and more consistent than install), but we can not assume the destroy steps are always the same for different cases. For better maintenance and easy reading, they should always be appeared as a pair.